Author: Adam Majer <amajer@suse.de>
Date: Fri Apr  6 10:59:49 CEST 2018
Bug: bsc#1084997
Summary: revert behaviour change since SLE11/SLE12

For package that tries to remove non-existent info page.

/sbin/install-info --quiet --delete --info-dir=/usr/share/info 
              /usr/share/info/blubber.info.gz ; echo $?
0


in SLE15,

/sbin/install-info --quiet --delete --info-dir=/usr/share/info 
              /usr/share/info/blubber.info.gz ; echo $?

install-info: No such file or directory for 
              /usr/share/info/blubber.info.gz 
1


So, we need to keep return code on failure to be 0 since suse_macros are missing
||: which results in broken packages that worked before to stop working now. This
is affecting upgrade process.

https://bugzilla.suse.com/show_bug.cgi?id=1084997

Index: texinfo-6.6/install-info/install-info.c
===================================================================
--- texinfo-6.6.orig/install-info/install-info.c
+++ texinfo-6.6/install-info/install-info.c
@@ -246,7 +246,7 @@ fatal (const char *fmt, ...)
   va_start (ap, fmt);
   vdiag (fmt, NULL, ap);
   va_end (ap);
-  exit (EXIT_FAILURE);
+  exit (EXIT_SUCCESS);
 }
 
 /* Return a newly-allocated string
